packages/dds/tree/src/test/shared-tree/sharedTree.bench.ts

```diff
@@ -389,6 +392,9 @@ describe("SharedTree benchmarks", () => {
 
 						// Setup
 						const provider = new TestTreeProviderLite(nbPeers, factory);
+						provider.trees.map((tree): void => {
+							tree.checkout.updateSchema(intoStoredSchema(jsonSequenceRootSchema));
+						});
```

Comment:
you don't need all trees to concurrently edit the schema.
```suggestion
						provider.trees[0].checkout.updateSchema(intoStoredSchema(jsonSequenceRootSchema));
						provider.processMessages();
```